packages/ariakit/src/heading/__examples__/heading/test.tsx

```diff
@@ -0,0 +1,31 @@
+import { getByRole, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render heading", () => {
+  render(<Example />);
+  expect(getByRole("heading-container")).toMatchInlineSnapshot(`
```

Comment:
```suggestion
  const { container } = render(<Example />);
  expect(container).toMatchInlineSnapshot(`
```

Reply:
Thanks for your suggestions @diegohaz , made all the changes, one final time(hopefully :))